packages/nameguard-react/src/components/Report/ReportFormattedName.tsx

```diff
@@ -1,26 +1,28 @@
 import React from "react";
+import { DisplayedName } from "../DisplayedName/DisplayedName";
+import { ENSName } from "@namehash/ens-utils";
+import { Normalization } from "@namehash/nameguard";
 
 type Props = {
+  name: ENSName;
   normalization: string;
-  name: string;
-  displayName: string;
+  beautifiedName?: string;
 };
 
 export function ReportFormattedDisplayName({
+  beautifiedName,
   normalization,
   name,
-  displayName,
 }: Props) {
-  const displayNameDifferent =
-    normalization === "normalized" && name !== displayName;
+  const displayNameDifferentThanBeautifiedName =
+    normalization === Normalization.normalized &&
+    (name.displayName !== beautifiedName || !beautifiedName);
```

Comment:
```suggestion
    (name.name !== beautifiedName || !beautifiedName);
```

The goal of the check here is if the displayName is different than the normalizedName